roles/pulp_common/README.md

```diff
@@ -289,7 +270,7 @@ Shared Variables
   Defaults to a list containing the single item "Jinja2" (which is necessary for pulp_installer to
 function). Also, a `prereq_role` may append to it.
 
-This role is required by the `pulp_database` role and uses some variables from it.
+* `pulp_settings.databases.default`: Documented above in the `pulp_settings` table as `databases.default`. This variable is shared with the [pulp_database](../../roles/pulp_database) role.
```

Comment:
have you checked if this link worked on mkdocs?

Reply:
I checked it, it does.